demo/src/App.tsx

```diff
@@ -21,19 +20,14 @@ let roomId: string;
   }
 })();
 
-interface AppProps {
-  client: ChatSdk;
-}
+interface AppProps {}
 
-const App: FC<AppProps> = ({ client }) => (
-  <RoomProvider
-    client={client}
-    roomId={roomId}
-  >
-    <div style={{ display: 'flex', justifyContent: 'space-between', width: '800px' }}>
+const App: FC<AppProps> = () => (
+  <RoomProvider roomId={roomId}>
+    <div style={{ display: 'flex', justifyContent: 'space-between', width: '800px', height: '800px' }}>
```

Comment:
Why the height?

Reply:
This was to stop the screen scrolling down and leaving the app half out of view